board/safety/safety_chrysler.h

```diff
@@ -1,18 +1,55 @@
 const int CHRYSLER_MAX_STEER = 261;
 const int CHRYSLER_MAX_RT_DELTA = 112;        // max delta torque allowed for real time checks
-const uint32_t CHRYSLER_RT_INTERVAL = 250000;  // 250ms between real time checks
-const int CHRYSLER_MAX_RATE_UP = 3;
-const int CHRYSLER_MAX_RATE_DOWN = 3;
+const uint32_t CHRYSLER_RT_INTERVAL = 250000; // 250ms between real time checks
+const int CHRYSLER_MAX_RATE_UP = 3;           //Must be double of limits set in op
+const int CHRYSLER_MAX_RATE_DOWN = 3;         //Must be double of limits set in op
 const int CHRYSLER_MAX_TORQUE_ERROR = 80;    // max torque cmd in excess of torque motor
-const int CHRYSLER_STANDSTILL_THRSLD = 10;  // about 1m/s
-const CanMsg CHRYSLER_TX_MSGS[] = {{571, 0, 3}, {658, 0, 6}, {678, 0, 8}};
+const int CHRYSLER_GAS_THRSLD = 7.7;          // 7% more than 2m/s changed from wheel rpm to km/h
```

Comment:
Might want to revert the speed signal change for this port, or verify it in another PR first for all cars